requirements_test.txt

```diff
@@ -1,5 +1,6 @@
 -r requirements_test_min.txt
+-r requirements_test_pre_commit.txt
```

Comment:
I just removed that in #978 because it makes a circular import where astroid need pylint 2.8.2 that need astroid 2.5.6 and break the tests

Reply:
Interesting... The pylint requirements should be fine in theory as it allows `>=2.5.6,<2.7` [pylint - setup.cfg](https://github.com/PyCQA/pylint/blob/091cb92314dc701f10390136da78fbbb362e892e/setup.cfg#L44)

Reply:
Found the issue: It seems to be an issue with `setuptools_scm` as `pip install -e .` would install version `0.1.dev1+...` https://github.com/PyCQA/astroid/pull/998/checks?check_run_id=2659259745#step:6:97

Reply:
The fix is to remove the pre-commit requirement from ``requirements_test.txt`` I don't know if it's a circular import but I think we cannot have both the local astroid and the dependency from pylint.

Reply:
The issue isn't a circular dependency. Somehow `pip install -e .` will install version `0.1.dev1` of astroid which is incompatible with `pylint`. I think that `setuptools_scm` doesn't get the correct version here.

Reply:
Found the issue: https://github.com/pypa/setuptools_scm/issues/414
Will look at solutions tomorrow.